### ISSUE TITLE
what about iris?

### ISSUE THREAD

**Issue by MohammadMirsafaei** [mingrammer/go-web-framework-stars#41]
https://github.com/kataras/iris
I think iris should be on list too

**Comment by ketloss** [comment]
Because [Iris is purposefully not included on awesome-go](https://github.com/avelino/awesome-go/pull/1135) and [some other reasons](https://github.com/julienschmidt/httprouter/issues/148).

I would recommend **anything** on the [above project](https://github.com/mingrammer/go-web-framework-stars) and would honestly prefer to write a web application in assembly language before using anything written by @katarus.  

He is a cancer to open source and it is only a matter of time before Microsoft cracks down [on this thing](https://www.reddit.com/r/golang/comments/b481q7/a_warning_about_githubcomkatarasiris/).

**Comment by vzool** [comment]
@ketloss Indeed, I got to use **Iris Framework** at the beginning when started my first web development using `golang` and it does behave weird which made me felt unsafe an vulnerable to all sort of danger.

**Comment by ABigNwa** [comment]
> @ketloss Indeed, I got to use **Iris Framework** at the beginning when started my first web development using `golang` and it does behave weird which made me felt unsafe an vulnerable to all sort of danger.

How so? Like aliens from space were out to get you? lol.
Its frustrating to search for information and read such nonsense all over the place.
